### ISSUE TITLE
Footer drop shadow

### ISSUE THREAD

**Issue by Feissit** [CUCentralAdvancement/essential-cu#347]
Please add a drop shadow to the footer

.rectangle {
  height: 403px;
  width: 1440px;
  background-color: #FFFFFF;
  box-shadow: 0 -3px 3px 0 rgba(0,0,0,0.22);
}

<img width="1474" alt="Screen Shot 2021-10-13 at 2 24 30 PM" src="https://user-images.githubusercontent.com/91230278/137207427-42b6e663-fe6a-419b-90fa-74eb8726d6b6.png">

**Comment by alexfinnarn** [comment]
<img width="573" alt="Screen Shot 2021-10-14 at 9 59 18 AM" src="https://user-images.githubusercontent.com/3640707/137343964-f6204356-4934-460e-8a9a-196f8a7bd0d7.png">

I can't really notice the shadow but it has been updated. Please advise.

**Comment by alexfinnarn** [comment]
I still can't really see the shadow with the InVision values or the Tailwind shadow-md. I did add a gold border at the top which I saw in the design. Let me know if you want further updates.

**Comment by duquennoisa** [comment]
@alexfinnarn not sure where you saw a gold border may have been an invision artifact. 

I can't see why the shadow isn't working either... I don't see one on the header either and I'm guessing its related. I see it in the code but it doesn't show up. Not sure what the issue is. Do you think it's a tailwind problem?

The footer is now linked to the credits issue #469

**Comment by duquennoisa** [comment]
I'm also seeing the same shadow issue with the story cards... nav bar buttons.

**Comment by alexfinnarn** [comment]
@duquennoisa The conflicting/missing button shadows were a mistake I made with duplicate class names on those elements. 

I now see shadows for all the elements you mentioned, except if the homepage story cards are supposed to have a shadow. It might not look like anything is there, but I definitely notice it when I remove the shadow via browser dev tools. Like I mentioned in #469, I don't mess around with the box-shadow values and instead, I've always used defaults. 

I think all the Tailwind shadows are directed downward and so it would be hard to see that on the footer unless there was an element below it. I know it's possible to "reverse" the shadow and direct it upwards, but you'd have to really get into the values to direct the shadow in any particular direction...there's also gradients that can look like an "inner shadow" and you can certainly combine a lot of techniques to emulate shadows and embossing styles.

However, I'm just going to put in there what people/Tailwind gives me and call it a day. Let me know if you have any specific values to add. Right now the scale is as follows:

```js
    boxShadow: {
      sm: '0 1px 2px 0 rgba(0, 0, 0, 0.05)',
      DEFAULT: '0 2px 4px 0 rgba(0,0,0,0.5)',
      bottom: '0 4px 4px -4px rgba(0,0,0,0.5)',
      liddle: '0 2px 6px 0 rgba(0,0,0,0.5)',
      md: '0 4px 6px -1px rgba(0, 0, 0, 0.1), 0 2px 4px -1px rgba(0, 0, 0, 0.06)',
      lg: '0 10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05)',
      xl: '0 20px 25px -5px rgba(0, 0, 0, 0.1), 0 10px 10px -5px rgba(0, 0, 0, 0.04)',
      inner: 'inset 0 2px 4px 0 rgba(0, 0, 0, 0.06)',
      none: 'none',
      footer: '0 -3px 3px 0 rgba(0,0,0,0.22)',
    },
```

**Comment by duquennoisa** [comment]
fixed with #469
